### PR TITLE
[nng] Update to v1.5.1

### PR DESCRIPTION
**Pull request by myd7349**
**Describe the pull request**

- #### What does your PR fix?  
  Update to v1.5.1.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All. Yes.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  <Yes / I am still working on this PR>

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/nng/portfile.cmake [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-    mbedtls NNG_ENABLE_TLS
-    tools NNG_ENABLE_NNGCAT
+    FEATURES
+        mbedtls NNG_ENABLE_TLS
+        tools NNG_ENABLE_NNGCAT
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (JonLiu1993) [inline ports/nng/portfile.cmake:17]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@myd7349, thanks for your PR, have you tested these features locally?

**Comment by myd7349** [comment]
> 
> 
> @myd7349, thanks for your PR, have you tested these features locally?

Hi! @JonLiu1993 Thanks for your help and sorry for the long delay.
I have tested all features with:
- x86-windows
- x86-windows-static
- x64-windows
- x64-windows-static

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the help!

**Pull request merged**